src/setup-beam.js

```diff
@@ -17,20 +17,24 @@ async function main() {
   const otpVersion = await installOTP(otpSpec, osVersion)
 
   const elixirSpec = core.getInput('elixir-version', { required: false })
-  const elixirInstalled = await maybeInstallElixir(elixirSpec, otpVersion)
+  const shouldMixHex = core.getInput('install-hex', {
```

Comment:
Moved here, since I was getting `install-hex: true` in the logs while it wasn't happening, so I wanted to make the default override explicit.